src/main/resources/org/jenkinsci/plugins/workflow/multibranch/Messages_fr.properties

```diff
@@ -0,0 +1,3 @@
+ReadTrustedStep._has_been_modified_in_an_untrusted_revis=\u2018{0}\u2019 a \u00e9t\u00e9 modifi\u00e9 dans une r\u00e9vision non fiable
+WorkflowMultiBranchProject.DisplayName=Pipeline Multibranches
+WorkflowMultiBranchProject.Description=Cr\u00e9e un ensemble de projets Pipeline en se basant sur les branches d\u00e9tect\u00e9es dans le d\u00e9p\u00f4t d'un gestionnaire de code source.
```

Comment:
Side note: give `.properties` are by definition encoded in `ISO-8859-1`, IMO it's probably unnecessary to encode using unicode entities. It is harder to review and hence easier to miss typos and the likes.

Thanks!